src/main/resources/js/script.js

```diff
@@ -0,0 +1,7 @@
+jQuery(function($){
```

Comment:
What is this??? You should not combine BE and FE implementations in one commit. If you want to add some FE changes, you should do it after implementation of your BE functionality